apps/rewards/app/components/Content/MyRewards.js

```diff
@@ -14,48 +16,59 @@ import {
   ONE_TIME_DIVIDEND,
   RECURRING_DIVIDEND,
   ONE_TIME_MERIT,
+  READY,
+  CLAIMED,
 } from '../../utils/constants'
 import { Empty } from '../Card'
 import Metrics from './Metrics'
-import { useAppState } from '@aragon/api-react'
+import { useAppState, useNetwork } from '@aragon/api-react'
 import BigNumber from 'bignumber.js'
-import { displayCurrency, getStatus } from '../../utils/helpers'
+import { displayCurrency, getStatus, getStatusId } from '../../utils/helpers'
 
 const MyRewards = ({
   myRewards,
   myMetrics,
   viewReward,
   claimReward,
+  claimHashes,
 }) => {
+  const network = useNetwork()
   const rewardsEmpty = myRewards.length === 0
 
+  const getEtherscanLink = reward => {
+    const networkChunk = network.id === 1 ? '' : `${network.type}.`
+    const claimHash = claimHashes[reward.rewardId]
+    const link = `https://${networkChunk}etherscan.io/tx/${claimHash}`
+    return link
```

Comment:
I like returning variables instead of complex expressions. It's easier to debug.